<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cpu_2</class>
 <widget class="QWidget" name="cpu_2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>259</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="root_box">
     <property name="title">
      <string>Процессор</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="file_box">
        <property name="title">
         <string>Файл прошивки</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <property name="horizontalSpacing">
          <number>6</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="fw_file_l">
           <property name="text">
            <string>Путь:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="fw_file_path">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QPushButton" name="select_fw_file">
           <property name="text">
            <string>Выбрать файл...</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="fw_file_size_l">
           <property name="text">
            <string>Размер:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="fw_file_size">
           <property name="maximum">
            <number>2147483647</number>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="fw_file_checksum_l">
           <property name="text">
            <string>Контрольная сумма:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="fw_file_checksum">
           <property name="text">
            <string>0x12345678</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QPushButton" name="reload_fw_file">
           <property name="text">
            <string>Перезагрузить файл</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="interface_box">
        <property name="title">
         <string>Интерфейс</string>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0" colspan="2">
          <widget class="QComboBox" name="interfaces"/>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QPushButton" name="update_interfaces">
           <property name="text">
            <string>Обновить список</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="bootloader_frame">
        <property name="title">
         <string>Бутлоадер</string>
        </property>
        <layout class="QFormLayout" name="formLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="btl_version_l">
           <property name="text">
            <string>Версия бутлодера:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="btl_version">
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="btl_fw_size_l">
           <property name="text">
            <string>Размер:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="btl_fw_size">
           <property name="maximum">
            <number>2147483647</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="btl_fw_checksum_l">
           <property name="text">
            <string>Контрольная сумма:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="btl_fw_checksum">
           <property name="text">
            <string>0x12345678</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="process_frame">
        <property name="title">
         <string>Процесс</string>
        </property>
        <layout class="QFormLayout" name="formLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="state_l">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Состояние:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="state">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Ожидание</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="progress_l">
           <property name="text">
            <string>Прогресс:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QProgressBar" name="progress">
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QPushButton" name="stop">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Принудительная остановка</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
